### PR TITLE
add generic perl recipes

### PR DESCRIPTION
**Pull request by jmairboeck**
I also added some explaining comments to the main recipe.

### CODE REVIEW

File: generic/generic_perl-1.23.recipe [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+LICENSE="MIT
+	GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://cpan.metacpan.org/authors/id/A/AU/AUTHOR/Project-X-$portVersion.tar.gz"
+CHECKSUM_SHA256="000000000000000000000000000000000"
+SOURCE_DIR="Project-X-$portVersion"
+PATCHES="project_x-$portVersion.patchset"
+ADDITIONAL_FILES="projectx.rdef"
+
+ARCHITECTURES="any"
```
Comment (jmairboeck) [inline generic/generic_perl-1.23.recipe:20]: Should we mark these with `?`? Most other generic recipes use it.

Comment (Begasus) [reply generic/generic_perl-1.23.recipe:20]: I didn't really change that on the other ones, so I guess for these it would be good to also use "?"

**Comment by Begasus** [comment]
Thanks, should check some perl packages with these. 👍

**Comment by jmairboeck** [comment]
Now the REQUIRES are sorted correctly, sorry for that.

**Pull request merged**